app/controllers/team.js

```diff
@@ -9,7 +9,7 @@ function inTeam(team) {
 }
 
 export default Controller.extend({
-  sortingKeys: Object.freeze(['added']),
+  sortingKeys: Object.freeze(['last', 'first']),
```

Comment:
I changed how we sort people. I thought it's more likely to see people sorted by their name in a website.